assets/src/styles/blocks/ArticlesEditor.scss

```diff
@@ -1,3 +1,33 @@
 .ignore-categories-wrapper {
   margin-top: 10px;
 }
+
+.articles-editor {
+  div.btn-secondary.article-load-more.btn.btn-block {
```

Comment:
Do we need the `div` and all classes in this selector for specificity? Would be nicer if it was just `.article-load-more`

Reply:
It was needed before to make sure that it wouldn't apply in the frontend, but now indeed since these styles are only loaded in the editor it's no longer necessary 👍